packages/ag-charts-community/src/integrated-charts-theme.ts

```diff
@@ -10,7 +9,13 @@ export * from './chart/themes/constants';
 export * from './chart/themes/util';
 export * from './module/theme';
 
-export const themes: Record<AgChartThemeName, AgChartTheme> = mapValues(
-    themeFactories,
-    (themeFactory) => themeFactory?.()!
-);
+export const themes: Record<AgChartThemeName, AgChartTheme> = {} as any;
```

Comment:
It wouldn't work to provide this as a function?

Reply:
Integrated probably needs to be able to iterate over the options - I just wanted to avoid changing the contract right now; we could probably easily change the values to be a function to simplify things here.